packages/runtime/plugin-runtime/package.json

```diff
@@ -130,6 +135,9 @@
       ],
       "router/server": [
         "./dist/types/router/runtime/server.d.ts"
+      ],
+      "loadable_bundler_plugin": [
```

Comment:
The "-" is used more in the repo. Can it be changed to the "-" here

Reply:
Got it, i update later

Reply:
Done~